### ISSUE TITLE
change playback speed 

### ISSUE THREAD

**Issue by syzo00** [aniyomiorg/aniyomi#1660]
### Describe your suggested feature

I'm thinking a feature when you hold the screen it turns the playback speed into 2x instead of the "save" feature because i think that everyone can screenshot it

### Other details

um I'm thinking about the feature like bilibili and loklok have (if you know that app)

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open or closed issue.
- [X] I have written a short but informative title.
- [X] If this is an issue with an official extension, I should be opening an issue in the [extensions repository](https://github.com/aniyomiorg/aniyomi-extensions/issues/new/choose).
- [X] I have updated the app to version **[0.15.3.0](https://github.com/aniyomiorg/aniyomi/releases/latest)**.
- [X] I will fill out all of the requested information in this form.

**Comment by littl3munkey** [comment]
Much needed feature imo

**Comment by Mishab25** [comment]
Yes, SyzOo i also requested to change into this. Hope they change in new update

**Comment by hungry-for-code** [comment]
Will they ever add this feature?
